source/v4/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -214,7 +214,7 @@ GOVUKFrontend.initAll()
 
 ### Select and initialise part of a page
 
-If you update a page with new markup, for example a modal dialogue box, you can run `initAll` with a `scope` parameter to initialise the components on part of a page.
+If you update a page with new markup, for example a modal dialogue box, you can run `initAll` with a `{ scope }` option to initialise the components on part of a page.
```

Comment:
I've clarified how the words "option", "parameter" and "property" are used

Looks like `initAll({ scope })` using options has always been the default?

Since running `initAll($scope)` as a parameter was only added in: https://github.com/alphagov/govuk-frontend/pull/6164